Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM golang:alpine AS builder
+
+RUN apk add --no-cache git
+
+RUN go install github.com/go-task/task/v3/cmd/task@latest
+
+FROM golang:alpine AS final
+
+LABEL org.opencontainers.image.authors="juanbarearojo0@correo.ugr.es"
+
+COPY --from=builder /go/bin/task /usr/local/bin/task
+
+RUN adduser -D -h /home/test test
+
+RUN mkdir -p /app/.cache/go-build \
+    && chmod -R 777 /app/.cache/go-build 
```

Comment:
Esto es un detalle tonto, pero, ¿has probado a intentar bajar los privilegios? 

En mi caso me funcionó con 747 en lugar de 777

Reply:
Se planteó usar pero si al usar 747. al ejecutar da este error
juan@juan-GF63-Thin-10UD ~/Escritorio/facultad/1_cuatri_cuarto/iv/privateChef $ docker run -u 1001 -t -v `pwd`:/app/test juanbarearojo0/privatechef
task: [test] go test ./internal/models/...
failed to initialize build cache at /app/.cache/go-build: mkdir /app/.cache/go-build/00: permission denied
task: Failed to run task "test": exit status 1. También se probó 755 o cambiar al usuario test antes de crear la carpeta pero ninguna funcionó. Por lo que se tuvo que recurrir a la 777